src/Classes/ServiceAPI/MyRadio_User.php

```diff
@@ -662,6 +669,32 @@ public function getOfficerships()
         return $this->officerships;
     }
 
+    /**
+     * Get the User's radio time, calculated from timeslot signins
+     */
+    public function getRadioTime()
+    {
+        if (!$this->radioTime) {
+            $this->radioTime = self::$db->fetchColumn(
```

Comment:
So... currently this will actually yeet an exception if the query fails. However, the code that does that is labeled `// TODO: temporary, uncomment this - marks.polakovs` - so normally it just returns an empty array, so this will explode - but with a cryptic error. Approving nonetheless, but consider fixing it :)

Reply:
shoved another commit with the catch, just said its null, unless we've got something to rethrow it to